components/patterns/form/_form.scss

```diff
@@ -0,0 +1,148 @@
+.section-wrapper--form {
+  max-width: 1024px;
```

Comment:
I saw your comment ` // NOTE: in a lot of this code, we're not using the rem() mixin, since we want the layout to be independent of text-zoom` in another file, so this might not be relevant but otherwise this could be in in the rem() mixin.